### PR TITLE
Update Base.hs

### PR DESCRIPTION
**Pull request by vaibhavsagar**
s/happend/happen/

Addresses #212.

### CODE REVIEW

File: Data/HashMap/Base.hs [hunk 1]
```diff
@@ -355,7 +355,7 @@ cmp cmpk cmpv t1 t2 = go (toList' t1 []) (toList' t2 [])
     go [] [] = EQ
     go [] _  = LT
     go _  [] = GT
-    go _ _ = error "cmp: Should never happend, toList' includes non Leaf / Collision"
+    go _ _ = error "cmp: Should never happened, toList' includes non Leaf / Collision"
```
Comment (rockbmb) [inline Data/HashMap/Base.hs]: Should it not be `happen` here instead of `happened`?

**Comment by vaibhavsagar** [comment]
Ping?

**Comment by treeowl** [comment]
Thanks.

**Comment by vaibhavsagar** [comment]
My pleasure!

**Pull request merged**